homework4/task2/tests/test_mock_input.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from project2_mock_input.task2_mock_input import count_dots_on_i
+
+
+def test_count_dots_on_i():
```

Comment:
What does this test check?

Reply:
It is an example from task description (in docstring). This test check that in page https://example.com/ there are 59 'i' chars.

Reply:
Yes, it is an example of the function.  Could your check the requirements for the test?

Reply:
ok, there is info about test with Mock only.